data/brands/amenity/post_office.json

```diff
@@ -332,6 +346,23 @@
         "brand:wikidata": "Q1815701"
       }
     },
+    {
+      "displayName": "Postes Canada",
+      "id": "postescanada-525cac",
+      "locationSet": {
+        "include": ["ca-qc.geojson"]
+      },
+      "matchNames": [
+        "canada post/ poste canada",
+        "poste canada",
+        "postes canada"
```

Comment:
proper spelling of the brand in French is Postes Canada, however there may be some cases misspelled as "Poste Canada"